src/contracts/pods/EigenPod.sol

```diff
@@ -55,6 +56,12 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
 
     /// @notice The amount of eth, in wei, that is restaked per ETH validator into EigenLayer
     uint256 public immutable REQUIRED_BALANCE_WEI;
+    
+
+    ///@notice The maximum amount of ETH, in gwei, a validator can have staked in the beacon chain
+    uint64 public immutable MAX_VALIDATOR_BALANCE_GWEI;
```

Comment:
this comment is presumably incorrect? validators can temporarily exceed 32 ETH but the excess will be automatically withdrawn, right? maybe we should just name this whatever is used in the spec itself?

Reply:
You're correct, this value represents the maximum amount of ETH you can stake not including earnings.  We use it in our hysteresis calculation:

min(32, floor(balance-0.75))